docs/pages/selectlist.js

```diff
@@ -267,8 +267,47 @@ export default function SelectListPage({ generatedDocGen }: {| generatedDocGen:
         <MainSection.Subsection
           title="Labels"
           description={`
-      SelectList comes with [Label](/label) built-in: just use the \`label\` prop. We strongly encourage always supplying a label. Be sure to provide a unique \`id\` so the Label is associated with the correct SelectList.`}
-        />
+      SelectList comes with [Label](/label) built-in: just use the \`label\` prop. We strongly encourage always supplying a label. Be sure to provide a unique \`id\` so the Label is associated with the correct SelectList.
+
+      If SelectList is labelled by content elsewhere on the page, or a more complex label is needed, the \`labelDisplay\` prop can be used to visually hide the label. In this case, it is still available to screen reader users, but will not appear visually on the screen.`}
```

Comment:
nit: technically for a US audience this should be "labeled" (with one "L")
https://www.grammarly.com/blog/labeled-labelled/
(I had to look this up 😅)